templates/emails/collective.update.published.hbs

```diff
@@ -35,7 +35,7 @@ Subject: {{update.title}}
 <br />
 
 <h2>{{update.title}}</h2>
-<div class="byline">by <a href="{{config.host.website}}/{{update.fromCollective.slug}}">{{update.fromCollective.name}}</a></div>
+<div class="byline">by <a href="{{config.host.website}}/{{collective.slug}}">{{collective.name}}</a></div>
```

Comment:
`fromCollective` and `collective` are not the same thing. `fromCollective` is the profile who submitted the update (so the user profile). `collective` is the profile where the update was posted.

I've fixed this behavior to include back the user name in my last commit.

Reply:
Oh! thanks :+1: